bsip-0086.md

```diff
@@ -0,0 +1,119 @@
+    BSIP: 0086
+    Title: Share market fees to the network
+    Author: Abit More <https://github.com/abitmore>
+    Status: Draft
+    Type: Protocol
+    Created: 2019-11-29
+    Discussion: https://github.com/bitshares/bsips/issues/194
+    Worker: TBD
+
+# Abstract
+
+This BSIP proposes a protocol change: when every trade happens, if there are
+market fees generated, a portion (E.G. 5% or 10%) of the market fees goes to
```

Comment:
e.g. 5% or 10% from 0.1%

Reply:
The whole sentence looks strange after the change. Any better idea?

Reply:
How about

"This BSIP proposes a protocol change: a portion (e.g. 5% or 10%) of every market trading fee (e.g. 0.1% of a filled trade) shall be shared with the network."

Reply:
"This BSIP proposes that a portion of every market trading fee shall be shared with the network."

Then add an example at the end of this paragraph.